ncm-network/src/main/perl/network.pm

```diff
@@ -279,7 +279,7 @@ sub Configure
                     $text .= "OVS_OPTIONS='$net{$iface}{ovs_opts}'\n";
```

Comment:
I think that is only required by `OVS_EXTRA` the rest of OVS options do not include variables. 
https://github.com/asteven/openvswitch/blob/master/rhel/README.RHEL